styles/_theme.scss

```diff
@@ -48,7 +48,7 @@ button,
 }
 
 .clrS {
-  background-color: $secondary;
+  background-color: #f0f0f0;
```

Comment:
@morebrownies what's going on here? The .clrS class is literally the "make this the color of $secondary" class. 

If the default secondary color should be different, the place to change that is in the _variables.scss file. 

To make it the same as #f0f0f0, change it to:

$secondary: adjust-color($primary, $lightness: -6%);